robottelo/ui/browser.py

```diff
@@ -109,6 +113,18 @@ def browser():
                 Ie() if settings.webdriver_binary is None
                 else Ie(executable_path=settings.webdriver_binary)
             )
+        elif webdriver_name == 'edge':
+            capabilities = webdriver.DesiredCapabilities.EDGE.copy()
+            capabilities['acceptSslCerts'] = True
```

Comment:
Seems as already fixed defect (at least for Firefox): https://bugzilla.mozilla.org/show_bug.cgi?id=1103196
Have you tried latest Selenium-WebDriver lib? Of course, we cannot use it in our project at that moment, but just for general interest